src/components/ChoosePlan.tsx

```diff
@@ -41,7 +20,7 @@ const ChoosePlan = () => {
         <span className='text-orange-400'>7 days free trial</span>
```

Comment:
get used to add all to a variables. Create styles.ts file and add colors.ts.

const colors = {
orange: '#...',
}

Reply:
I will do it when I convert from `taildwindCSS` to `styled-components`